test/support/presenter_test_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module PresenterTestHelpers
+  def it_includes_last_edited_by_editor_id
+    should "[:last_edited_by_editor_id]" do
+      editor = FactoryBot.create(:user)
+      edition.actions.create!(request_type: Action::CREATE, requester: editor)
+
+      assert_equal editor.uid, result[:last_edited_by_editor_id]
+    end
+  end
+end
```

Comment:
We should include test that ensures us we are actually sending last editor id, not id of the user who created the edition. Although it seems like we do not have access to this information atm.

Reply:
Cool, I've updated the tests to cover both eventualities. Looking at the behaviour of the code, the only time a `CREATE` action is created is when creating a brand new edition (i.e one without any children), if a new edition is created from an existing edition, the creates a `NEW_VERSION` action.